webservices/templates/swagger-ui.html

```diff
@@ -4,116 +4,127 @@
 <head>
   <title>OpenFEC API Documentation</title>
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700|Source+Code+Pro:300,600|Titillium+Web:400,600,700" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="{{ swagger_static('dist/main.css') }}" >
-    <link rel="icon" type="image/png" href="{{ swagger_static('favicon.ico') }}" />
-    <style>
-      html
-      {
-        box-sizing: border-box;
-        overflow: -moz-scrollbars-vertical;
-        overflow-y: scroll;
-      }
-
-      *,
-      *:before,
-      *:after
-      {
-        box-sizing: inherit;
-      }
-
-      body
-      {
-      margin: 0;
-      background: #fafafa;
-      }
-      
-      label[for=user_first_name]:before
-      {
-      content: " *";
-      color: #E87723;
-      font-weight: 1000;
-      font-family: "Verdana";
-      }
+  <link rel="stylesheet" type="text/css" href="{{ swagger_static('dist/main.css') }}" >
+  <link rel="icon" type="image/png" href="{{ swagger_static('favicon.ico') }}" />
+  <style>
+    html
+    {
+      box-sizing: border-box;
+      overflow: -moz-scrollbars-vertical;
+      overflow-y: scroll;
+    }
+
+    *,
+    *:before,
+    *:after
+    {
+      box-sizing: inherit;
+    }
+
+    body
+    {
+    margin: 0;
+    background: #fafafa;
+    }
+    
+    label[for=user_first_name]:before
+    {
+    content: " *";
+    color: #E87723;
+    font-weight: 1000;
+    font-family: "Verdana";
+    }
+
+    label[for=user_last_name]:before
+    {
+    content: " *";
+    color: #E87723;
+    font-weight: 1000;
+    font-family: "Verdana";
+    }
+    
+    label[for=user_email]:before
+    {
+    content: " *";
+    color: #E87723;
+    font-weight: 1000;
+    font-family: "Verdana";
+    }
+
+    #signup_head
+    {
+    font-size: 24px;
+    }
+
+  </style>
+</head>
 
-      label[for=user_last_name]:before
-      {
-      content: " *";
-      color: #E87723;
-      font-weight: 1000;
-      font-family: "Verdana";
-      }
+<body>
+  <div id="swagger-ui"></div>
+
+  <script type="text/javascript">
+    /* {#
```

Comment:
The diff here is weird, too. For these, I didn't like all the comments being sent to the page so I made them source comments (yml, Jinja, handlebars, whichever) so we can read them but so they don't go into the page source.